### PR TITLE
Added in code to detec slack and strip out braces and url

### PR DESCRIPTION
**Pull request by Rudikza**

### CODE REVIEW

File: lib/lita/handlers/down_for_everyone.rb [hunk 1]
```diff
@@ -1,4 +1,5 @@
 require "lita"
+require "pry"
```
Comment (killpack) [inline lib/lita/handlers/down_for_everyone.rb]: Would you mind removing this dependency? It doesn't look like it's actually used anywhere (though I bet you used it when you were debugging).

File: spec/lita/handlers/down_for_everyone_spec.rb [hunk 2]
```diff
@@ -1,8 +1,8 @@
 require "spec_helper"
 
 describe Lita::Handlers::DownForEveryone, lita_handler: true do
-  it { routes_command("is heroku.com down").to :is_site_down }
-  it { routes_command("is github.com down again").to :is_site_down }
+  it { is_expected.to route_command("is heroku.com down").to :is_site_down }
+  it { is_expected.to route_command("is github.com down again").to :is_site_down }
```
Comment (killpack) [inline spec/lita/handlers/down_for_everyone_spec.rb]: Would you mind adding a test case for this changeset, too?

**Comment by killpack** [comment]
Mostly looks good-- just a couple small housekeeping details. Thanks for the PR!

File: spec/lita/handlers/down_for_everyone_spec.rb (hunk 2 above)
Comment (Rudikza) [inline spec/lita/handlers/down_for_everyone_spec.rb]: Done.

**Comment by Rudikza** [comment]
@killpack spec added and pry has been removed.

**Comment by killpack** [comment]
Thanks again!

**Comment by Rudikza** [comment]
No worries, I see that they are trying to fix this problem in the lita-slack adapter because many other plugins also have issues. https://github.com/kenjij/lita-slack/pull/35

**Pull request merged**